src/lang/Messages.properties

```diff
@@ -803,7 +803,14 @@ cfu.warn.addOnSameVersion = The same version of the add-on is already installed:
 cfu.warn.addOnNotRunnable.message = The add-on will not run until the following requirements are fulfilled:
 cfu.warn.addOnNotRunnable.question = Continue with the installation?
 cfu.warn.cantload      = Can not load the specified add-on\:\nNot before \= {0}\nNot from \= {1}
-cfu.warn.invalidAddOn = The selected file is not a valid ZAP add-on.
+cfu.warn.invalidAddOn = The selected file is not a valid ZAP add-on{0}
+cfu.warn.invalidAddOn.noName = :\nThe file has no name.
```

Comment:
What's this mean? How can you select a file with no name?

Reply:
That happens when you try to validate/create an add-on with a null or root path (though not through the UI which just allows to select files). I guess the message could be changed to the path is not valid instead.

Reply:
That seems more meaningful

Reply:
Changed.